src/go/rdctl/cmd/root.go

```diff
@@ -61,27 +62,41 @@ func Execute() {
 }
 
 func init() {
-  var err error
+	var err error
 
-  cobra.OnInitialize(initConfig)
-  configDir, err = os.UserConfigDir()
-  if err != nil {
-    log.Fatal("Can't get config-dir: ", err)
-  }
-  defaultConfigPath = filepath.Join(configDir, "rancher-desktop", "rd-engine.json")
-  rootCmd.PersistentFlags().StringVar(&configPath, "config-path", "", fmt.Sprintf("config file (default %s)", defaultConfigPath))
-  rootCmd.PersistentFlags().StringVar(&user, "user", "", "overrides the user setting in the config file")
-  rootCmd.PersistentFlags().StringVar(&host, "host", "", "default is localhost; most useful for WSL")
-  rootCmd.PersistentFlags().StringVar(&port, "port", "", "overrides the port setting in the config file")
-  rootCmd.PersistentFlags().StringVar(&password, "password", "", "overrides the password setting in the config file")
+	cobra.OnInitialize(initConfig)
+	configDir, err = os.UserConfigDir()
+	if err != nil {
+		log.Fatal("Can't get config-dir: ", err)
+	}
+	defaultConfigPath = filepath.Join(configDir, "rancher-desktop", "rd-engine.json")
+	rootCmd.PersistentFlags().StringVar(&configPath, "config-path", "", fmt.Sprintf("config file (default %s)", defaultConfigPath))
+	rootCmd.PersistentFlags().StringVar(&user, "user", "", "overrides the user setting in the config file")
+	rootCmd.PersistentFlags().StringVar(&host, "host", "", "default is localhost; most useful for WSL")
+	rootCmd.PersistentFlags().StringVar(&port, "port", "", "overrides the port setting in the config file")
+	rootCmd.PersistentFlags().StringVar(&password, "password", "", "overrides the password setting in the config file")
 }
 
-func doRequest(method string, command string)  error {
-  req, err := getRequestObject(method, command)
-  if err != nil {
-    return err
-  }
-  return doRestOfRequest(req)
+var emptyByteArray []byte
```

Comment:
Why are you using this constant instead of returning `nil` directly?

Reply:
Because of that `if len(result) > 0 {` code, which is now gone.